src/database/utilities/utilities.js

```diff
@@ -20,3 +21,44 @@ export function addBatchToParent(batch, parent, createItem) {
   if (item.batches && item.batches.find(currentBatch => currentBatch.id === batch.id)) return;
   item.addBatch(batch);
 }
+/**
+ * Removes TransactionBatch...
```

Comment:
* 1 - Remove ItemBatch from TransactionBatch
 * 2 - Remove TransactionBatch from TransactionItem
 * 3 - if TransactionItem has no batches, remove TransactionItem from Transaction

Above as per full function comment.  Had to ... the end of first line since explanation is not a one liner